src/vs/platform/terminal/common/capabilities/capabilities.ts

```diff
@@ -9,6 +9,8 @@ import type { IPromptInputModel, ISerializedPromptInputModel } from './commandDe
 import { ICurrentPartialCommand } from './commandDetection/terminalCommand.js';
 import { ITerminalOutputMatch, ITerminalOutputMatcher } from '../terminal.js';
 import { ReplayEntry } from '../terminalProcess.js';
+// eslint-disable-next-line local/code-import-patterns
+import { TerminalShellIntegrationEnvironment } from 'vscode';
```

Comment:
We can't import from vscode here, you can just duplicate this interface below `export interface IShellEnvDetectionCapability`